src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserManagementResource.kt

```diff
@@ -35,6 +35,49 @@ class UserManagementResource(
   private val userService: UserService,
 ) {
 
+  @PreAuthorize("hasRole('ROLE_MANAGE_NOMIS_USER_ACCOUNT')")
+  @PutMapping("/{username}/logon-date")
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "Record the date and time of last logon for the user",
```

Comment:
Are you recording the date and time of the last logon for the user - or are you recording a new logon?  Have they just tried to sign in and was it successful?  Thinking more along the lines of
```suggestion
    summary = "Record a user as having successfully signed in, with current date and time written to the LAST_LOGON_DATE",
```